src/blueblock/Human.java

```diff
@@ -11,29 +11,29 @@
 			new Color(155, 0, 155) };
 	private Label[][] labels = Main.labels;
 	private int pos1, pos2, Dpos1, Dpos2;
-	public int Kills, schritte, SuperScore, PowerUps, Spieler;
-	private boolean lives, poisoned, gesch�tzt;
+	public int Kills, steps, SuperScore, PowerUps, Player;
+	private boolean lives, poisoned, guarded;
```

Comment:
I just noticed `protected` would fit more ("guared" is more like "bewacht" than "beschützt", I think).